docs/content/getting-started/hello-dagster.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Hello Dagster | Dagster
+description: Run dagster for the first time
+---
+
+# `hello dagster`
+
+This guide helps you build a simple data pipeline in dagster that downloads the top 10 HackerNews stories.
+
+1. Start by creating a file called `hello-dagster.py` and copying the following:
+
+    ```python file=/getting-started/hello-dagster/hello-dagster.py
+    # copy to a file called hello-dagster.py
+    from dagster import repository, asset, MetadataValue
+    import requests
+    import pandas as pd
+
+    # dagster helps you think about datasets, models, 
+    # and other objects that you want to exist as assets
+    # instead of worrying about tasks
+
+    @asset
+    def hackernews_topstory_ids():
+        """
+        Get up to 500 top stories from the HackerNews topstories endpoint.
+        API Docs: https://github.com/HackerNews/API#new-top-and-best-stories
+        """
+        newstories_url = "https://hacker-news.firebaseio.com/v0/topstories.json"
+        top_500_newstories = requests.get(newstories_url).json()
+        return top_500_newstories
+
+    # dependencies between assets can be expressed as function arguments
+    @asset 
+    def hackernews_topstories(context, hackernews_topstory_ids):
+        """
+        Get items based on story ids from the HackerNews items endpoint
+        """
+
+        results = []
+        for item_id in hackernews_topstory_ids:
+            item = requests.get(f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json").json()
+            results.append(item)
+            if len(results) >= 10:
+                break
+
+        df = pd.DataFrame(results)
+
+        # dagster tracks rich metadata about our assets
+        context.add_output_metadata(
```

Comment:
We typically recommend adding output metadata using 
```
return Output(
    value=df,
    metadata={...},
)
```

Reply:
is this documented anywhere? i have seen this pattern before but i don't think it's in our api docs anywhere

Reply:
also - i wonder if we should just skip including metadata as part of the example. seems like 2 extra concepts (what's `Output`? what's `metadata`?) that we may want to introduce more gradually

Reply:
The documentation on output metadata is definitely in need of some love. I included it in this example so that when someone runs the pipeline they can see the hackernews data somewhere. Otherwise it is tricky to validate that dagster did anything without determining how to load the asset from a pickle file in an ugly tmp directory.